apps/service/package.json

```diff
@@ -27,8 +28,9 @@
     "@nestjs/platform-fastify": "^10.3.9",
     "convert-keys": "^1.3.4",
     "json-rpc-2.0": "^1.7.0",
+    "lodash": "^4.17.21",
     "reflect-metadata": "^0.2.0",
-    "rgbpp": "^0.5.0",
+    "rgbpp": "0.0.0-snap-20240813134030",
```

Comment:
> May I ask why the `.js` suffix is needed? It worked fine on my local computer without `type: module` and the `.js` suffix.

@duanyytop, it was due to the upgrade of `rgbpp`. It works fine with version `0.5.0`, but the following error is thrown when upgraded to `0.0.0-snap-20240813134030` (a relatively new version of the snapshots). Do you have a clue on how to resolve the issue without specifying `"type": "module"` in the package.json?

```
/Users/root/Development/projects/rgbpp-sdk/node_modules/.pnpm/@rgbpp-sdk+ckb@0.0.0-snap-20240813134030_@ckb-lumos+lumos@0.22.0-next.5_lodash@4.17.21/node_modules/@rgbpp-sdk/ckb/dist/index.js:469
var import_ckb_sdk_core = __toESM(require("@nervosnetwork/ckb-sdk-core"));
                                  ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/root/Development/projects/rgbpp-sdk/node_modules/.pnpm/@nervosnetwork+ckb-sdk-core@0.109.2/node_modules/@nervosnetwork/ckb-sdk-core/lib/index.js from /Users/root/Development/projects/rgbpp-sdk/node_modules/.pnpm/@rgbpp-sdk+ckb@0.0.0-snap-20240813134030_@ckb-lumos+lumos@0.22.0-next.5_lodash@4.17.21/node_modules/@rgbpp-sdk/ckb/dist/index.js not supported.
/Users/root/Development/projects/rgbpp-sdk/node_modules/.pnpm/@nervosnetwork+ckb-sdk-core@0.109.2/node_modules/@nervosnetwork/ckb-sdk-core/lib/index.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead either rename /Users/root/Development/projects/rgbpp-sdk/node_modules/.pnpm/@nervosnetwork+ckb-sdk-core@0.109.2/node_modules/@nervosnetwork/ckb-sdk-core/lib/index.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/root/Development/projects/rgbpp-sdk/node_modules/.pnpm/@nervosnetwork+ckb-sdk-core@0.109.2/node_modules/@nervosnetwork/ckb-sdk-core/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```

Reply:
I retried on my local computer, `.js` suffix is required.